dataland-batch-manager/src/main/kotlin/org/dataland/datalandbatchmanager/service/CompanyUploader.kt

```diff
@@ -30,32 +30,38 @@ class CompanyUploader(
     private val logger = LoggerFactory.getLogger(javaClass)
 
     @Suppress("ReturnCount")
```

Comment:
It is difficult to make the code leaner because of the hierarchical structure of the response. 
Each consequent step depends on all of the prior steps, so it appears difficult to me to write this in a leaner way